regexp-diff.md

```diff
@@ -0,0 +1,31 @@
+# RegExp changes
+
+```diff
+21.2.3.1 RegExp ( pattern, flags )
+
+The following steps are taken:
+
+-    1. Let patternIsRegExp be ? IsRegExp(pattern).
```

Comment:
Pending the discussion in #2 and #4 I'm not sure if this is accurate. What do you think about what we should do for RegExps?

Reply:
I think this has been resolved. I will remove this file.